### DISCUSSION TITLE
does resetting password update LDAP database?

### DISCUSSION THREAD

**Discussion by itsyigid** [authelia/authelia#8155] [Q&A] [answered]
Hi, the title should be enough. I tested the LDAP backend (OpenLDAP) and I think Authelia doesn't update the database, but I have to be sure.

Does resetting the password update LDAP backend's database? If not, why?

**Comment by james-d-elliott** [comment, marked as answer]
Yes and it's a tested interaction.

**Reply by itsyigid** [reply]
thank you so much!!
